MediaGalleryUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -286,30 +286,30 @@
         }
 
         .image-details {
+            display: block;
+
             .image-details-image {
-                float: left;
-                max-width: 70%;
-                width: auto;
+                max-width: 500px;
 
                 img {
-                    max-height: 650px;
+                    max-height: 450px;
```

Comment:
Hi @gabrieldagama , this 450px is intended for portrait images when reaching tablet screen size or smaller. I had to decrease the max-height so that when the image and details stack up the image title will be visible above the fold of the screen. 650px still works for 768 and above screen sizes.
![image](https://user-images.githubusercontent.com/47961795/85854035-381ce400-b7e6-11ea-8e43-4fd5b4068426.png)